README.md

```diff
@@ -12,7 +12,7 @@ We have a "changelog" of decisions: [CHANGELOG.md](CHANGELOG.md).
 
 These updates provide high level information about the Editorial Board's activities:
 
-- [2024-01](content/updates/2024-01-08-editorial-board-update.md)
+- [Updates](https://python.github.io/editorial-board/updates/)
```

Comment:
Rethinking this, perhaps it's better to embed the link in the previous sentence.

Eg

Replace 
> These updates provide high level information about the Editorial Board's activities:
with
Updates of the Editorial Board's activities can be read [here](https://python.github.io/editorial-board/updates/).
Subscribe to the updates using the [RSS Feed](https://python.github.io/editorial-board/updates/index.xml).

Reply:
Updated, although I reworded the first one to:
```md
[Read updates](https://python.github.io/editorial-board/updates/) of the Editorial Board's activities can be read.
```
Because for accessibility:
> * Avoid link text like “Click Here,” “More,” and “Read More.” These kinds of links can be confusing when a screen reader reads them out of context.

https://usability.yale.edu/web-accessibility/articles/links

How does that look?

Reply:
Sounds good. Thanks!